modules/profile/manifests/jenkins/agent.pp

```diff
@@ -100,9 +100,9 @@
   include docker
 
   # required by cleanup_docker script
-  python::pip { 'docker-py':
-    pkgname        => 'docker-py',
-    #ensure        => present, # defaults to present
+  python::pip { 'docker':
+    ensure  => '2.5.0',
```

Comment:
A 2.5.0 release happened between the start of this adventure and this morning. It's what I got when I ran `pip3 install docker` on the test hosts so that's what I'm pinning for future hosts.